packages/twenty-front/src/modules/navigation/hooks/__tests__/useIsMockedDrawerPage.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { renderHook } from '@testing-library/react';
+import { RecoilRoot } from 'recoil';
+
+import { useIsMockedDrawerPage } from '@/navigation/hooks/useIsMockedDrawerPage';
+import { AppPath } from '@/types/AppPath';
+import { useIsMatchingLocation } from '~/hooks/useIsMatchingLocation';
+
+jest.mock('~/hooks/useIsMatchingLocation');
+const mockUseIsMatchingLocation = jest.mocked(useIsMatchingLocation);
+
+const setupMockIsMatchingLocation = (pathname: string) => {
```

Comment:
I think a better strategy could be to let the prefetch not that it's loaded in the case of the mocked app (see my comment below)

So I would say that we don't need this hook at all

Reply:
ok, done